libraries/radspreadprocessing/working-with-rows-and-columns/resizing.md

```diff
@@ -236,3 +237,30 @@ __Example 8__ shows how to convert and set from MS Excel to pixel column width.
 {{endregion}}
 
 The row height in MS Excel is measured in points so in order to set them you can convert this unit and set the exact number you are passing to the **SetHeight**() method for the height using the UnitHelper class.
+
+
+## LayoutHelper class
```

Comment:
Add link to the API reference page for this class. It could be somewhere in the content of this section and in the See Also section.